L1Trigger/L1TTwinMux/src/DTRPCBxCorrection.cc

```diff
@@ -44,11 +44,11 @@ void DTRPCBxCorrection::BxCorrection(int track_seg){
   L1MuDTChambPhDigi * dtts=0;
   L1MuDTChambPhDigi * dttsnew=0;
   L1MuDTChambPhDigi * rpcts1=0;
-  std::auto_ptr<L1MuTMChambPhContainer> m_phiDTDigis_tm(new L1MuTMChambPhContainer);
+  std::shared_ptr<L1MuTMChambPhContainer> m_phiDTDigis_tm(new L1MuTMChambPhContainer);
```

Comment:
This could also be done with a std::make_shared

Reply:
Hi Thomas,
Ok I did that. Thanks